spec.html

```diff
@@ -8393,7 +8393,7 @@ <h1>Static Semantics: TopLevelVarScopedDeclarations ( ): a List of Parse Nodes</
       </emu-alg>
       <emu-grammar>LabelledItem : Statement</emu-grammar>
       <emu-alg>
-        1. If |Statement| is <emu-grammar>Statement : LabelledStatement</emu-grammar> , return TopLevelVarScopedDeclarations of |Statement|.
+        1. If |Statement| is <emu-grammar>Statement : LabelledStatement</emu-grammar>, return TopLevelVarScopedDeclarations of |Statement|.
```

Comment:
There are 6 remaining cases that you might like to handle similarly. Search for `> ,` and `> .` (3 of each).